Marlin/src/MarlinCore.cpp

```diff
@@ -1053,6 +1053,10 @@ void setup() {
   #if HAS_FILAMENT_SENSOR
     SETUP_RUN(runout.setup());
   #endif
+  
+  #if HAS_TMC220x
+    SETUP_RUN(tmc_serial_begin());
+  #endif
```

Comment:
Does this have to occur ahead of the PSU control? As I understand it, TMC stepper drivers require 12V before they can be configured. But perhaps this doesn't matter to the initial serial connection here.